Source/hydro/Castro_advection_1d.F90

```diff
@@ -211,16 +211,16 @@ subroutine umeth(q, q_lo, q_hi, &
        ! limiting, and returns the integral of each profile under each
        ! wave to each interface
        do n = 1, NQ
-          call ppm_reconstruct(q(:,n), q_lo, q_hi, &
+          call ppm_reconstruct(q, q_lo, q_hi, NQ, n, &
                                flatn, q_lo, q_hi, &
                                sxm, sxp, q_lo, q_hi, &
                                lo(1), 0, hi(1), 0, dx, 0, 0)
 
-          call ppm_int_profile(q(:,n), q_lo, q_hi, &
-                               q(:,QU), q_lo, q_hi, &
-                               qaux(:,QC), qa_lo, qa_hi, &
+          call ppm_int_profile(q, q_lo, q_hi, NQ, n, &
+                               q, q_lo, q_hi, &
```

Comment:
Can you simplify this interface a little bit? Now we're just passing q twice which is redundant.

Reply:
Nevermind, ignore my comment